quilt/tools/package.py

```diff
@@ -161,8 +161,16 @@ def save_df(self, df, name, path, ext, target):
 
         # Serialize DataFrame to chosen format
         if enumformat is PackageFormat.HDF5:
-            with pd.HDFStore(storepath, mode='w') as store:
-                store[self.DF_NAME] = df
+            # HACK: Force the use of old pickle to ensure Python 2/3 compatibility.
+            from pandas.compat import cPickle
+            old_protocol = cPickle.HIGHEST_PROTOCOL
```

Comment:
since goal is to restore old_protocol, shouldn't we get it from pandas if possible? e.g. this will break if their source code changes.